soc/espressif/esp32c6/CMakeLists.txt

```diff
@@ -71,6 +88,17 @@ endif()
 
 if(CONFIG_MCUBOOT)
   set(SOC_LINKER_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/mcuboot.ld CACHE INTERNAL "")
+elseif(CONFIG_SOC_ESP32C6_LPCORE)
+  set(SOC_LINKER_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/default_lpcore.ld CACHE INTERNAL "")
+  set(CMAKE_C_FLAGS "-Os -march=rv32imac__zicsr_zifencei -mdiv -fdata-sections -ffunction-sections"
+    CACHE STRING "C Compiler Base Flags")
+set(CMAKE_CXX_FLAGS "-Os -march=rv32imac__zicsr_zifencei -mdiv -fdata-sections -ffunction-sections"
```

Comment:
wonky indent and don't see why these flags should even be here @tejlmand

Reply:
This is some test leftovers, thanks for spotting them!